{"uuid":"2edf0511-5811-4522-90fd-56e088549244","historyId":"aebf7b4c2e4dc0df746bea010c982f6b","fullName":"testcases.placeOrderTest.testRegisterwithValidCredentials","labels":[{"name":"package","value":"testcases.placeOrderTest"},{"name":"testClass","value":"testcases.placeOrderTest"},{"name":"testMethod","value":"testRegisterwithValidCredentials"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"testcases.placeOrderTest"},{"name":"host","value":"DESKTOP-63AKE56"},{"name":"thread","value":"16492@DESKTOP-63AKE56.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"testRegisterwithValidCredentials","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"\n[public void testcases.placeOrderTest.testRegisterwithValidCredentials(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.lang.InterruptedException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).\nData provider mismatch\nMethod: testRegisterwithValidCredentials([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2, java.lang.String arg3, java.lang.String arg4, java.lang.String arg5, java.lang.String arg6, java.lang.String arg7, java.lang.String arg8, java.lang.String arg9, java.lang.String arg10, java.lang.String arg11])\nArguments: [(java.lang.String) Ronny,(java.lang.String) Rohan,(java.lang.String) Ronny@gmail.com,(java.lang.String) Democratic People's Republic of Korea,(java.lang.String) Port Edris,(java.lang.String) 4612 Reginald Tunnel,(java.lang.String) 65480-6451,(com.github.javafaker.PhoneNumber) com.github.javafaker.PhoneNumber@4451f60c,(java.lang.String) Ronny Rohan,(com.github.javafaker.PhoneNumber) com.github.javafaker.PhoneNumber@4451f60c,(java.lang.String) 1234,(java.lang.String) Your order has been successfully processed!]","trace":"org.testng.internal.reflect.MethodMatcherException: \n[public void testcases.placeOrderTest.testRegisterwithValidCredentials(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.lang.InterruptedException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).\nData provider mismatch\nMethod: testRegisterwithValidCredentials([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2, java.lang.String arg3, java.lang.String arg4, java.lang.String arg5, java.lang.String arg6, java.lang.String arg7, java.lang.String arg8, java.lang.String arg9, java.lang.String arg10, java.lang.String arg11])\nArguments: [(java.lang.String) Ronny,(java.lang.String) Rohan,(java.lang.String) Ronny@gmail.com,(java.lang.String) Democratic People's Republic of Korea,(java.lang.String) Port Edris,(java.lang.String) 4612 Reginald Tunnel,(java.lang.String) 65480-6451,(com.github.javafaker.PhoneNumber) com.github.javafaker.PhoneNumber@4451f60c,(java.lang.String) Ronny Rohan,(com.github.javafaker.PhoneNumber) com.github.javafaker.PhoneNumber@4451f60c,(java.lang.String) 1234,(java.lang.String) Your order has been successfully processed!]\r\n\tat org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)\r\n\tat org.testng.internal.Parameters.injectParameters(Parameters.java:909)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:37)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1713540004834,"stop":1713540004840}